src/test/resources/company-profile-enumMapper-expected-output.json

```diff
@@ -46,8 +46,8 @@
     "external_registration_number" : "12345",
     "foreign_company_details" : {
         "accounting_requirement" : {
-            "foreign_account_type" : "1",
-            "terms_of_account_publication" : null
+            "foreign_account_type" : null,
+            "terms_of_account_publication" : "something"
```

Comment:
shouldn't this be mapped to one of the string values in the mapper?

Reply:
Ah I fiddled with that for a bit while testing, because that field needed to be populated for the account type test to work, and that was before I started work on the terms field so I didn't know what to put yet. I ended up not using that file for the terms field test so forgot to change it to a legit string. Have changed now!